include/swift/AST/Decl.h

```diff
@@ -4455,15 +4452,20 @@ class AbstractStorageDecl : public ValueDecl {
     Bits.AbstractStorageDecl.SupportsMutation = implInfo.supportsMutation();
   }
 
+  struct {
+    unsigned IsGetterMutatingComputed : 1;
+    unsigned IsGetterMutating : 1;
+    unsigned IsSetterMutatingComputed : 1;
+    unsigned IsSetterMutating : 1;
+  } LazySemanticInfo = { };
```

Comment:
What makes you prefer to have these fields in `LazySemanticInfo` than in the `Bits`?

Reply:
It means I can put the 'friend class FooRequest;' inside AbstractStorageDecl and not ValueDecl. It doesn't really matter though, and putting it in Bits would save memory.

@DougGregor any thoughts?